napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -293,13 +293,48 @@ def _set_table(self, layer_str: str = ''):
             self._table.setItem(0, 0, item)
 
     def _get_layer_actions(self):
+        """
+        Get all the actions that could potencially raise a keybinding conflict
+        taking into account the current selected keybinding group.
+
+        Returns
+        -------
+        actions_all: tuple
```

Comment:
```suggestion
        actions_all: tuple[str, tuple[str, Action]]
```
hopefully this is correct.